js/wNumb.js

```diff
@@ -23,12 +23,12 @@
 var FormatOptions = [
 	'decimals',
 	'thousand',
-	'mark',
-	'prefix',
-	'suffix',
-	'encoder',
-	'decoder',
-	'negativeBefore',
+	"mark",
+	"prefix",
+	"suffix",
+	"encoder",
+	"decoder",
+	"negativeBefore",
 	'negative',
 	'edit',
 	'undo'

```

Comment:
Any reason to mix single and double quotes?

Reply:
Nope. Actually that's a reason to hold off on merger. I'll be cleaning up the singles that are left, that was an oversight!